examples/annotation_label_studio/steps/convert_annotation_step.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from typing import Any, Dict, List
+
+from zenml.logger import get_logger
+from zenml.steps import Output, step
+
+logger = get_logger(__name__)
+
+
+@step
+def convert_annotation(
```

Comment:
is the convert function something specific to this example only or it could be moved to some built-in function instead of implementing it as part of the example

Reply:
Unfortunately, due to the way that Label Studio outputs their annotations, (and how they import them), there will always need to be some kind of conversion step. I've been saying a bunch on Discord that I'd love it if ZenML was the one to build a universal conversion tool between all the formats, but I think for this PR that's too much :)